toolset/benchmark/fortune_html_parser.py

```diff
@@ -74,20 +74,30 @@ def handle_charref(self, name):
     # Not sure why some are escaping '/'
     if val == "47" or val == "047" or val == "x2f":
       self.body.append("/")
+    # "&#40;" is a valid escaping of "(", but
+    # it is not required, so we need to normalize for out
+    # final parse and equality check.
+    if val == "40" or val == "040" or val == "x28":
```

Comment:
Note that there's an identical failure in the build for #2175 

This: https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/151374139#L2664-L2680
#2175: https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/151376090#L5173-L5189

Or:

This: https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/151374134#L3068-L3087
#2175: https://s3.amazonaws.com/archive.travis-ci.org/jobs/151376071/log.txt (Travis log view is corrupted here for me)